guides/README.md

```diff
@@ -0,0 +1,16 @@
+# Guides for leto-modelizer-plugin-core
+
+## Docs
+
+In this [docs section](https://github.com/ditrit/leto-modelizer-plugin-core/blob/main/guides/docs/guide.md), you will learn how plugin-core works overall and will have a detailed view of all the classes and methods you will find in this project.
```

Comment:
rework on it and bad link
add explanation on essentials with link on it
add explanation on configuration with link on it

Reply:
done